featurex/converters/api.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, api_key=None):
         self.recognizer = sr.Recognizer()
         self.api_key = api_key
 
+    @abstractmethod
```

Comment:
That seems reasonable. Or maybe you could check inside `get_converter` if the class has a defined, non-`None` `environ_key` attribute.